.babelrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          ie: '11'
```

Comment:
I guess there's already a target on `package.json` too, remove one of them

Reply:
Yeah, That's true, Thanks.